src/qt/guiutil.cpp

```diff
@@ -764,10 +764,10 @@ QString NetworkToQString(Network net)
     assert(false);
 }
 
-QString ConnectionTypeToQString(ConnectionType conn_type)
+QString ConnectionTypeToQString(ConnectionType conn_type, bool relay_txes)
 {
     switch (conn_type) {
-    case ConnectionType::INBOUND: return QObject::tr("Inbound");
+    case ConnectionType::INBOUND: return QObject::tr(relay_txes ? "Inbound Full Relay" : "Inbound Block Relay");
```

Comment:
df5fb7ed64f5e47cad66c2fe3582c66a7896f818, for correct work of translation mechanics
```suggestion
    case ConnectionType::INBOUND: return relay_txes ? QObject::tr("Inbound Full Relay") : QObject::tr("Inbound Block Relay");
```

Reply:
It seems equivalent? Anyway, done.

Reply:
It seems equivalent? Anyway, done--thank you for looking!

Reply:
It is [not equivalent](https://doc.qt.io/qt-5/i18n-source-translation.html#using-tr-to-obtain-a-translation) for the `lupdate` tool:
> When Qt's translation tool, lupdate, is used to process a set of source files, the text wrapped in `tr()` calls is stored in a section of the translation file that corresponds to its translation context.